src/explorer/siteNodeBase.ts

```diff
@@ -258,4 +249,23 @@ export class SiteNodeBase extends NodeBase {
         const serverFarmId = util.parseAzureResourceId(this.site.serverFarmId.toLowerCase());
         return await this.webSiteClient.appServicePlans.get(serverFarmId.resourcegroups, serverFarmId.serverfarms);
     }
+
+    protected async convertScmType(node: SiteNodeBase, config: WebSiteModels.SiteConfigResource): Promise<void> {
```

Comment:
nit: private instead of protected

Also I like `updateScmType` better than `convertScmType` (it matches `updateConfiguration` below)

Reply:
I like that better too! I was thinking about adding an info window that alarms the scmType is being updated even if it was originally none. Thoughts?

Reply:
Are you talking about removing the check on line 259 so that the message always displays? If so, sounds good to me

If you're talking about a separate message, than maybe consider the output window instead. I don't want two messages popping up within seconds